tests/testthat/test-capture-requests.R

```diff
@@ -1,6 +1,8 @@
 d <- tempfile()
 dl_file <- tempfile()
 webp_file <- tempfile()
+file_path <- tempfile()
+writeLines(letters[1:6], file_path)
```

Comment:
The windows issue is almost surely about CRLF being inserted here. From `?writeLines`:

     Normally ‘writeLines’ is used with a text-mode connection, and the
     default separator is converted to the normal separator for that
     platform (LF on Unix/Linux, CRLF on Windows).  For more control,
     open a binary connection and specify the precise value you want
     written to the file in ‘sep’.  For even more control, use
     ‘writeChar’ on a binary connection.

I found a related test for this area in a different file, and it just uses `cat()` (with no newlines) to write to a tempfile: https://github.com/nealrichardson/httptest2/blob/main/tests/testthat/test-mock-api.R#L165

It would also be good to extend that test block ("POST with all body types") to cover the strings and raw, to make sure the request assertions work as expected.

Reply:
Done and it solves the problem in windows